tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -58,31 +66,41 @@ prepare: |
     snap download --channel=20/edge pc
     unsquashfs -d gadget-dir pc_*.snap
 
+execute: |
+    # this test simulates a reinstall, to clean the TPM this requires
+    # a reboot so the losetup has to be redone
+    if [ "$SPREAD_REBOOT" = 1 ]; then
+        ./losetup.sh
+    fi
+    LOOP="$(cat loop.txt)"
+
+    echo "Setup simulated ubuntu-seed mount"
+    mkdir -p /run/mnt/ubuntu-seed
+    mount "${LOOP}"p2 /run/mnt/ubuntu-seed
+
     echo "Install EFI binaries"
     bootdir=/run/mnt/ubuntu-boot/EFI/boot
     mkdir -p "$bootdir"
     cp /usr/lib/shim/shimx64.efi.signed "$bootdir"/bootx64.efi
     cp /usr/lib/grub/x86_64-efi-signed/grubx64.efi.signed "$bootdir"/grubx64.efi
-
-execute: |
-    bootdir=/run/mnt/ubuntu-boot/EFI/boot
     ls -l "$bootdir"
     sbverify --list "$bootdir"/bootx64.efi
     sbverify --list "$bootdir"/grubx64.efi
-    LOOP="$(cat loop.txt)"
 
     echo "Run the snap-bootstrap tool"
     go get ../../lib/uc20-create-partitions
     uc20-create-partitions \
-        --encrypt --key-file keyfile \
-        --recovery-key-file recovery-key \
-        --policy-update-data-file policy-update-data \
-        --tpm-lockout-auth tpm-lockout-auth \
+        --encrypt --key-file /run/mnt/ubuntu-seed/keyfile \
+        --recovery-key-file /run/mnt/ubuntu-data/system-data/var/lib/snapd/fde/recovery-key \
```

Comment:
it's really device/fde in the real code, not sure it matters here

Reply:
Thanks, fixed as well. It does not really matter but I would prefer it to be as close as possible to the real thing :)